pegasus/sites.v3/hourofcode.com/public/css/action-blocks.scss

```diff
@@ -110,40 +109,10 @@
   }
 
   @media screen and (max-width: $width-sm) {
-    a.link-button, button {
+    a.link-button,
+    button {
       width: 100%;
       text-align: center;
     }
   }
 }
-
-// Editorial cards
-// See code.org/teach and code.org/help for examples
-.editorial-cards-wrapper {
-  display: grid;
-  margin-top: 4rem;
-  gap: 2.5rem;
-
-  @media screen and (min-width: $width-sm) {
-    grid-template-columns: repeat(3, 1fr);
-  }
-
-  .flex-container {
-    gap: 1.5rem;
-    flex-direction: column;
-
-    &.self-paced-english {
```

Comment:
I was going to say- we've removed a lot of styling files for .self-paced-english but only one reference to it!

Reply:
I know right! I discovered a better way to do this w/o needing specificity, hence this entire PR 😂